pytorch_transformers/tokenization_xlnet.py

```diff
@@ -177,6 +177,16 @@ def convert_tokens_to_string(self, tokens):
         out_string = ''.join(tokens).replace(SPIECE_UNDERLINE, ' ').strip()
         return out_string
 
+    def add_special_tokens_single_sentence(self, token_ids):
+        logger.warning("No method was defined for special tokens and single sentence streams in XLNet. "
```

Comment:
So the general way to define these is to look how the pretraining is made (in this case it's like a sentence pair pretraining) and adapt it in the closest way to single sentence.
In the case of XLNet I would thus go with `sep + token_ids[1] + sep + cls` I guess

Reply:
Seeing as the sentence pair has `A [SEP] B [SEP][CLS]`, shouldn't it be `B [SEP][CLS]` for the single sentence one?